eth/downloader/queue.go

```diff
@@ -39,8 +39,8 @@ import (
 )
 
 var (
-	blockCacheLimit   = 8192 // Maximum number of blocks to cache before throttling the download
-	maxInFlightStates = 4096 // Maximum number of state downloads to allow concurrently
+	blockCacheLimit   = 16384 // Maximum number of blocks to cache before throttling the download
```

Comment:
Ok, lets reduce the block cache back to 8K, but I'd really like to keep the states up. Especially when we have stalling peers (you know which ones), it clogs up the download on entire subtries so I'd really like to increase the concurrency there. Trie nodes are mostly tiny anyway, measuring on average 300 bytes.